website/content/docs/api-clients/client-agent.mdx

```diff
@@ -521,6 +521,8 @@ Follow the troubleshooting steps to understand why the Client Agent is not able
 On MacOS versions 15.1 and 15.2, the firewall may incorrectly block the Client Agent from sending DNS responses. To resolve this issue,
 upgrade to MacOS version 15.3 or later.
 
+It may be neccessary to explicitly allow the `boundary-client-agent` process access to incoming network connections.
```

Comment:
I would add a little more detail on where to enable this setting

```suggestion
It may be neccessary to explicitly allow the `boundary-client-agent` process access to incoming network connections.

From the Firewall options in System Settings, toggle the `boundary-client-agent' to **Allow incoming connections**.  
```